### DISCUSSION TITLE
Aggiornamento Supervisor 2022.10.2 impossibile

### DISCUSSION THREAD

**Discussion by centurion78** [indomus/forum#2055] [HUB: Home Assistant]
Buongiorno a tutti,
ieri sera ho provato ad effettuare vari aggiornamenti del mio HA tra cui quello del Supervisor alla 2022.10.2, ma la cosa mi è impossibile. Quando clicco su installa mi appare il seguente messaggio: "invocazione del servizio  update/install non riuscita. Error updating Home assistant supervisor". La cosa assurda è che non mi carica nulla, ho cercato un po' di informazioni online e ho trovato che si può tentare il comando "ha supervisor update" da CLI, ma non mi carica più il terminale. Se ci clicco su mi appare il messaggio "Terminal & SSH
Add-on is not running. Please start it first", vado e premo AVVIA ed il messaggio è "Impossibile avviare il componente aggiuntivo
Unknown error, see supervisor". E questo mi viene restituito per qualunque componente aggiuntivo. Qualcuno di voi sa come uscire da questa empasse? Avevo pensato di collegarmi tramite SSH, ma se non mi avvia il componente non ci posso accedere da remoto. 
Avevo anche altri aggiornamenti in sospeso: Home Assistant Core 20.11.1 e Home Assistant Operating System 9.3, ma non posso avviarli, il messaggio in entrambi i casi è "invocazione del servizio update/install non riuscita. Error updatin home assistant operating system. 'OSManager.update blocked from execution, supervisor needs to be updated first".
Spero che qualcuno possa darmi una mano.

**Comment by centurion78** [comment]
Ah, mi ero dimenticato di scrivere che se faccio riavviare HA non si riavvia. Ho provato ad inserire la microSD sia in un PC Win10 che in Ubuntu ma non si può leggere. Volevo far apparire la scheda Supervisor, ma senza l'editor non posso accedere al file configuration.yaml

**Comment by Dim-1** [comment]
A me il tuo errore ogni tanto me lo ha dato, e non ne conosco il motivo. Per fare l'update in questo caso, mi basta riavviare l'host, cioè Debian che ospita il mio home assistant supervised su docker.  Il comando per riavviare l'host lo trovi tra le chiamate di servizio in strumenti per sviluppatori, oppure lo lanci via ssh. Però non ho capito che tipo di installazione hai.

**Reply by centurion78** [reply]
Scusa Dim-1, 
Intanto grazie per la risposta, mi sono scordato di scrivere le info riguardo alla mia attuale configurazione:
         Home Assistant 2022.10.5
         Supervisor 2022.10.0
         Operating System 9.2
         Frontend 20221010.0 - latest
Al momento lavora su di un RaspberryPi 4 a 64b e ho installato direttamente Home Assistant OS.
Purtroppo non riesco a collegarmi via SSH. Con Putty mi restituisce "Network Error: Connection Refused". La cosa non mi stupisce in quanto non si avvia "Terminal & SSH" e se ci vado su mi da il solito errore "Add-on is not running. Please start it first", al tentativo di avvio mi restituisce l'errore "Impossibile avviare il componente aggiuntivo - Unknown error, see supervisor"

**Reply by centurion78** [reply]
Ho provato, a fatica, a lanciare il servizio "hassio.host_reboot", si è riavviato ma è tutto come prima.

**Reply by Dim-1** [reply]
Purtroppo non so aiutarti, mi spiace. Fai ricerche il inglese, di solito si trovano molte più cose

**Reply by centurion78** [reply]
Grazie mille. Avevo già cercato su google in inglese ma nulla. Ora sto provando sul forum di HA in lingua inglese. Ho postato proprio 10 minuti fa.
